### PR TITLE
multiplayer reactions keyboard ui updates from testing

### PR DESCRIPTION
**Pull request by jwunderl**
Feedback in testing this am was that reactions pane looked a little worse with all the keyboard hint, so took a stab at rearranging to hopefully look a little more inviting again.

* make the numbers on the emojis look a little 'like keys' as some weren't sure that users would understand the circled numbers:
image

* move from a text description under the reactions to the 'keyboard controls dialog'
image

took the same icon as the one on the reactions button to hopefully guide them to press that if they see it and haven't noticed the reactions yet.

@unthinkmedia let me know what you think~

### CODE REVIEW

File: multiplayer/src/components/KeyboardControlsInfo.tsx [hunk 1]
```diff
@@ -32,6 +34,14 @@ export default function Render() {
                     </div>
                 );
             })}
+            <div className="keymap-row tw-flex tw-flex-row tw-items-center">
+                <div className="keymap-key tw-align-middle tw-min-w-[2.5rem] tw-text-center tw-pb-[2.1rem]">
+                    1-6
+                </div>
+                <span className="keymap-name tw-font-semibold tw-text-center">
+                    <ReactionsIcon className="hover:tw-scale-125 tw-ease-linear tw-duration-[50ms]" />
```
Comment (thsparks) [inline multiplayer/src/components/KeyboardControlsInfo.tsx:42]: So does this actually work if they click on it?

File: multiplayer/src/components/KeyboardControlsInfo.tsx [hunk 2]
```diff
@@ -32,6 +34,14 @@ export default function Render() {
                     </div>
                 );
             })}
+            <div className="keymap-row tw-flex tw-flex-row tw-items-center">
+                <div className="keymap-key tw-align-middle tw-min-w-[2.5rem] tw-text-center tw-pb-[2.1rem]">
+                    1-6
```
Comment (thsparks) [inline multiplayer/src/components/KeyboardControlsInfo.tsx:39]: nit, either way is fine, but personally it'd have a slight preference for spaces around the dash (1 - 6).

**Review by eanders-ms — APPROVED** [review]

**Review by thsparks — APPROVED** [review]
LGTM just some minor comments

File: multiplayer/src/components/KeyboardControlsInfo.tsx (hunk 1 above)
Comment (jwunderl) [reply multiplayer/src/components/KeyboardControlsInfo.tsx]: No, this one's just the icon that's used as a label for the button in `Reactions.tsx`, not the button itself (I did forget to save after removing the hover state though, so removed that~)

**Pull request merged**